tests/integrations/huggingface/cassettes/test_huggingface_inference_client/test_huggingface_document_question_answering_async.yaml

```
@@ -1,4 +1,357 @@
+      X-Frame-Options:
+      - deny
+      X-GitHub-Request-Id:
+      - 8844:31126C:45375D:5BB6F0:67B4DD87
+      X-Served-By:
+      - cache-bom4743-BOM
+      X-Timer:
+      - S1739988962.834505,VS0,VE0
+      X-XSS-Protection:
+      - 1; mode=block
+    status:
+      code: 200
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)